apps/system/style/chrome/chrome.css

```diff
@@ -386,6 +392,8 @@
 .controls .back-button[disabled],
 .controls .forward-button[disabled] {
   display: none;
+  opacity: 0;
+  animation: fadeOut .2s ease-in;
```

Comment:
Would a visibility transition work here, or do we need a display? If visibility is sufficient, we could do it in a single transition rule, using the transition delay. If not, perhaps animation is our only bet or using more javascript. 

Also how do we control the fade out *before* setting the display to 0 currently? I'll try to look into this as well.